From a23b7bd59bba5269e2a00dbbf50e519ca0373651 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Tue, 10 Jan 2017 01:05:30 +0100
Subject: [PATCH 3/7] Use <var> instead of <em>

<emphasis> is not allowed within <literal> in DocBook, which is what
this gets converted to. Instead use <var> which is allowed, and is also
a more correct markup anyway.
---
 policy.sgml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index afa05a6..084472c 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -1920,8 +1920,8 @@ zope.
 	  It must start with the line <tt>#!/usr/bin/make -f</tt>,
 	  so that it can be invoked by saying its name rather than
 	  invoking <prgn>make</prgn> explicitly. That is, invoking
-          either of <tt>make -f debian/rules <em>args...</em></tt>
-          or <tt>./debian/rules <em>args...</em></tt> must result in
+          either of <tt>make -f debian/rules <var>args...</var></tt>
+          or <tt>./debian/rules <var>args...</var></tt> must result in
           identical behavior.
 	</p>
 
-- 
2.11.0

